server/planning/search/queries/planning.py

```diff
@@ -270,7 +270,16 @@ def search_coverage_assignment_status(params: Dict[str, Any], query: elastic.Ela
             query.must_not.append(
                 elastic.nested(
                     path="coverages",
-                    query=elastic.bool_query(must=[elastic.exists(field="coverages.assigned_to.assignment_id")]),
+                    query=elastic.bool_query(
```

Comment:
This change is not related to Coverage Desk validation, and would be better suited to a separate PR

Reply:
okay I revert these changes and create a separate PR for this.